hawkmoth/parser.py

```diff
@@ -170,6 +170,8 @@ def _decl_fixup(ttype, name):
 
     ttype, name = _function_pointer_fixup(ttype, name)
 
+    if ttype == '_Bool': ttype = 'bool'
```

Comment:
I think it's useful to follow the style we have for other fixups. One function per fixup, and the comments for the function can desribe what it's about.